index.ejs

```diff
@@ -1,108 +1,106 @@
-<!DOCTYPE html>
-<!--[if lt IE 7 ]> <html lang="en" class="ie6" <% if(htmlWebpackPlugin.files.manifest) { %> manifest="<%= htmlWebpackPlugin.files.manifest %>"<% } %>> <![endif]-->
-<!--[if IE 7 ]>    <html lang="en" class="ie7" <% if(htmlWebpackPlugin.files.manifest) { %> manifest="<%= htmlWebpackPlugin.files.manifest %>"<% } %>> <![endif]-->
-<!--[if IE 8 ]>    <html lang="en" class="ie8" <% if(htmlWebpackPlugin.files.manifest) { %> manifest="<%= htmlWebpackPlugin.files.manifest %>"<% } %>> <![endif]-->
-<!--[if IE 9 ]>    <html lang="en" class="ie9" <% if(htmlWebpackPlugin.files.manifest) { %> manifest="<%= htmlWebpackPlugin.files.manifest %>"<% } %>> <![endif]-->
-<!--[if (gt IE 9)|!(IE)]><!--> <html lang="en" class="" <% if(htmlWebpackPlugin.files.manifest) { %> manifest="<%= htmlWebpackPlugin.files.manifest %>"<% } %>> <!--<![endif]-->
-<head>
-  <meta charset="utf-8">
-  <% if (htmlWebpackPlugin.options.meta) { %>
-  <% for (var key in htmlWebpackPlugin.options.meta) { %>
-  <meta name="<%= key %>" content="<%= htmlWebpackPlugin.options.meta[key] %>">
-  <% } %>
-  <% } %>
-
-  <title><%= htmlWebpackPlugin.options.title || 'Webpack App'%></title>
-
-  <% if (htmlWebpackPlugin.files.favicon) { %>
-  <link rel="shortcut icon" href="<%= htmlWebpackPlugin.files.favicon%>">
-  <% } %>
-  <% if (htmlWebpackPlugin.options.mobile) { %>
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <% } %>
+<% var item, key %>
```

Comment:
This change potentially makes the internals of the template easier to read by itself.  But it introduces a level of abstraction in the template that I kind of feel like avoiding.  I would like to come to this template and see immediately how it works.  If there are if statements and template writes that map straight back to webpack values, there is little to understand.  If there is now an intermediate variables that rename values assigned to them, I have to understand that layer as well.  I'd prefer to leave this out.